### PR TITLE
New feature that allows for selecting rows that don't match a given s…

### PR DESCRIPTION
**Pull request by rogerlindberg**
…tring.

### CODE REVIEW

File: rlselect.py [hunk 1]
```diff
@@ -99,6 +99,19 @@ def match(line):
...
         marks = set()
         for term, term_len in terms:
+            # Negative matching..
+            if term[0] == '!':
+                # A single exclamation char always matches
+                if term == '!':
+                    continue
+                # A double exclamation string means match for single exclamation char
+                elif term == '!!':
+                    term = '!'
```
Comment (rickardlindberg) [inline rlselect.py:109]: Check if term starts with double !!, then remove one so that it can match "!any"?

Comment (rogerlindberg) [reply rlselect.py:109]: Yes, that's how it works.

Comment (rickardlindberg) [reply rlselect.py:109]: Example input:
```
foo
!bar
```
Search term: 
```
!!b
```
Expected filtered:
```
!bar
```
Actual filtered:
```
foo
```
Shouldn't the if be something like
```python
elif term.startswith("!!"):
    term = term[1:]
```

Comment (rogerlindberg) [reply rlselect.py:109]: Yes, of course, you are absolutely right.

**Pull request merged**